src/Stepper/Step.less

```diff
@@ -0,0 +1,171 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+@stepIconWidth: 1.75rem; // 28px
+@iconFocusShadowThickness: @size_2xs; // 4px
+@titleFontSize: 1.25rem; // 20px
+@alertSize: 0.375rem; // 6px
+
+.Step {
+  .flexbox();
+  .alignItems(flex-start);
+  .padding--y--2xs;
```

Comment:
just fyi; no need to change this right now, since we need to do a mass change in this repo anyway, but this syntax for mixins is getting deprecated, IIRC.
need to start always including the parentheses now: `.padding--y--2xs()`, etc